### ISSUE TITLE
Getting a Error as "ERROR: type "abstime" does not exist" when executing BPS DB scripts against Postgres 14.1

### ISSUE THREAD

**Issue by ShanikaWickramasinghe** [wso2/product-is#12970]
**How to reproduce:**
1. Configure postgres latest version (14.1) with a docker image
2. Create a  database and exectute the db postgres db scripts agaianst postgres 14.1
3. When execute /dbscripts/bps/bpel/create/postgresql.sql script getting below errors

**Note**
Didnt observe this issue against postres 11.5 . Got this against postgres latest version (14.1)
![Screenshot from 2022-01-12 12-16-23](https://user-images.githubusercontent.com/31848014/149078114-b9907831-79e2-4d78-83b8-06a5605158d3.png)

```
SQL Error [42704]: ERROR: type "abstime" does not exist
  Position: 131
  ERROR: type "abstime" does not exist
  Position: 131
  ERROR: type "abstime" does not exist
  Position: 131
```

```
2022-01-12 12:16:16.314 - org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [42704]: ERROR: type "abstime" does not exist
  Position: 131
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:513)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$0(SQLQueryJob.java:444)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:171)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:431)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.run(SQLQueryJob.java:220)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: org.postgresql.util.PSQLException: ERROR: type "abstime" does not exist
  Position: 131
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2553)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2285)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:323)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:481)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:401)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:322)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:308)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:284)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:279)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:130)
	... 7 more

```

Related https://github.com/wso2/product-is/issues/10414

**Environment**
5.12.0 alpha 9
postgres version (14.1 (Debian 14.1-1.pgdg110+1))

**Comment by anuragpathak21** [comment]
This issue was also reported in earlier version and was closed stating unable to preproduce in 5.11 beta   #7343 .  It's back in the latest version. I wonder how.

**Comment by ShanikaWickramasinghe** [comment]
@anuragpathak21 Did you check with the latest version ( 5.12.0) against postgres 14.1. #7343 was reported against postgres 12.1. This was observed against postgres 14.1

**Comment by nilasini** [comment]
This issue has been fixed with [1]. The issue has been fixed from carbon-process 4.5.49 onwards but wso2is-5.12.0-alpha9 contains 4.5.47 therefore the issue was reproduced but with latest pack the issue has been fixed.

[1] https://github.com/wso2/carbon-business-process/pull/640
